models/ios/ntp_global/ios_ntp_global.yaml

```diff
@@ -0,0 +1,228 @@
+module: ios_ntp_global
+short_description: ntp_global resource module
+description:
+- This module provides declarative management of ntp on Cisco IOS devices.
+version_added: 2.5.0
+author:
+- Sagar Paul (@KB-perByte)
+notes:
+- Tested against Cisco IOSv Version 15.6.
+- This module works with connection C(network_cli).
+options:
+  config:
+    description: A dictionary of ntp options
+    elements: dict
+    suboptions:
+      access_group:
+        description: Control NTP access
+        type: dict
+        suboptions:
+          ipv4: 
+            description:  ipv4 access lists (Default not idempotent)
+            type: bool
+          ipv6: 
+            description:  ipv6 access lists (Default not idempotent)
+            type: bool
+          peer: 
+            description: Provide full access
+            type: dict
+            suboptions:
+              access_list: &access_group_acl
+                description:  name or number of access list
+                type: str
+              kod: &kod
+                description: Send a Kiss-o-Death packet for failing peers
+                type: bool
+          query-only: 
+            description:  Allow only control queries
+            type: dict
+            suboptions: 
+              access_list: *access_group_acl
+              kod: *kod
+          serve: 
+            description:  Provide server and query access
+            type: dict
+            suboptions:
+              access_list: *access_group_acl
+              kod: *kod
+          serve-only: 
+            description:  Provide only server access
+            type: dict
+            suboptions:
+              access_list: *access_group_acl
+              kod: *kod
+      allow:
+        description: Allow processing of packets
+        type: dict
+        elements: dict
+        suboptions:
+          control:
+            description: Allow processing control mode packets
+            type: bool
+          private:
+            description: Allow processing private mode packets
+            type: bool
+          rate_limit_delay:
+            description: Rate limiting delays
+            type: int
+      authenticate:
+        description: Authenticate time sources
+        type: bool
+      authentication_key:
```

Comment:
Please match the key names as per https://github.com/ansible-network/resource_module_models/pull/149/files#diff-f6fc3bc94c48edafdd7ee275b4dacb2553a99690c1edb5721a395df58691274aR67. This has been previously followed for OSPF interfaces too.